3.9/upgrading-community-to-enterprise.md

```diff
@@ -28,6 +28,9 @@ and installation of the Enterprise package. The upgrade can be done in a
 description of the two methods. Refer to the sections below for a detailed
 procedure.
 
+The Enterprise Edition of ArangoDB requires a license to activate the Enterprise Edition features.
```

Comment:
Technically it is not just about activating the features, but running EE (i.e., an EE without a license key does not just become a community version, but does not run at all)